api/src/Modules/volunteers/Controller/Hours/GetDepartmentHoursSummary.php

```diff
@@ -87,14 +80,20 @@ public function buildResource(Request $request, Response $response, $params): ar
         }
 
         $sum = 0;
+        $volunteers = [];
+        $vol_count = 0;
         foreach ($data as $entry) {
             $sum += $entry['total_hours'];
+            if (!in_array($entry['member'], $volunteers)) {
+                $vol_count++;
```

Comment:
Would it make more sense to calculate the count based on the length of the final `$volunteers` array? 

I might also be missing it, but where is `$volunteers` used here? Is that part of a base class?

Reply:
Agreeded. Updated.